features/toolbar/components/Toolbar.js

```diff
@@ -1,10 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import {
-    APP_SCREEN,
-    navigate
-} from '../../app';
+import { navigate } from '../../base/navigation';
+import { WELCOME_SCREEN } from '../../welcome';
```

Comment:
I know we don't want to import things from non-base features, but I'm not seeing a better option that doesn't lead to defining all of the screens inside the base directory.